numpy/lib/index_tricks.py

```diff
@@ -631,7 +631,7 @@ class ndindex:
 
     Examples
     --------
-    # dimensions as individual arguments
+    >>> # dimensions as individual arguments
```

Comment:
```suggestion
    Dimensions as individual arguments
    
```

Reply:
I think my suggestion renders correctly?

Reply:
yes your suggestions also renders correctly and seems cleaner, let me commit those suggestions.